src/nyc_trees/apps/event/models.py

```diff
@@ -58,9 +61,16 @@ def has_space_available(self):
         return self.eventregistration_set.count() < self.max_attendees
 
     @property
-    def starting_soon(self):
-        # TODO: Implement
-        return True
+    def starting_soon_from_now(self):
+        return self.starting_soon(timezone.now())
+
+    def starting_soon(self, dt):
```

Comment:
I think this api would be a little cleaner as:
```python
# it's nice to put config-related, arbitrary values up top.
_STARTING_SOON_WINDOW = timedelta(hours=1)

# ...

# This way you don't need a separate method that exists just for testing.
def starting_soon(self, seed_dt=None):
    seed_dt = seed_dt or timezone.now()
    return (seed_dt >= self.begins_at - _STARTING_SOON_WINDOW and
               seed_dt < self.ends_at)
```

This shouldn't require changes to any of the other files, other than names.